devito/ops/transformer.py

```diff
@@ -37,7 +42,44 @@ def opsit(trees, count):
         parameters
     )
 
-    return ops_kernel
+    stencil_arrays_initializations = itertools.chain(*[
+        to_ops_stencil(p, node_factory.ops_args_accesses[p])
+        for p in parameters if isinstance(p, OpsAccessible)
+    ])
+
+    pre_time_loop = stencil_arrays_initializations
+
+    return pre_time_loop, ops_kernel
+
+
+def to_ops_stencil(param, accesses):
+    dims = len(accesses[0])
+    pts = len(accesses)
+    stencil_name = "s%sd_%s_%spt" % (dims, param.name, pts)
+
+    stencil_array = SymbolicArray(
+        name=stencil_name,
+        dimensions=(pts * dims,),
+        dtype=np.int32
+    )
+
+    ops_stencil = OpsStencil(stencil_name.upper())
+
+    return [
+        Expression(ClusterizedEq(Eq(
+            stencil_array,
+            ListInitializer(list(itertools.chain(*accesses)))
+        ))),
+        Expression(ClusterizedEq(Eq(
+            ops_stencil,
+            ExternalFunctionCall("ops_decl_stencil", [
```

Comment:
namespace['ops_decl_stencil']

Reply:
Are we expecting the API to change?

Reply:
And actually, even then, this is only used in one place. There's no need to take it out

Reply:
Shouldn't we add all OPS conventions?
like [that](https://github.com/opesci/devito/blob/5008e36a3b2ba268e257cc9fcdc91b48c170ac3b/devito/yask/utils.py#L66)

Reply:
I think it should go in the namespace too.

Although it's not beautiful, it's what we do with all other names (same in YASK), so for homogeneity I  think it's better this way